at_secondary/at_persistence_secondary_server/lib/src/log/commitlog/at_commit_log.dart

```diff
@@ -32,7 +33,9 @@ class AtCommitLog implements AtLogType {
     // So return -1.
     // The private: and privatekey: are not synced. so return -1.
     if (!key.startsWith('public:__') &&
-        key.startsWith(RegExp('private:|privatekey:|public:_'))) {
+        (key.startsWith(RegExp(
+                'private:|privatekey:|public:_|public:signing_publickey')) ||
+            key.contains(AT_SIGNING_PRIVATE_KEY))) {
```

Comment:
Use of `contains` seems potentially buggy. Why not an exact match?

Reply:
format of the key is <current_atsign>:signing_privatekey<current_atsign>

Reply:
key.startsWith(RegExp('@(?<sharedWith>.*):signing_privatekey@(?<sharedBy>.*)'))
is this fine?

Reply:
LGTM